### PR TITLE
remove unnecessary lets

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
@westy92 this PR removes any use of let that isn't required, and replaces it with const.

File: src/index.ts [hunk 1]
```diff
@@ -36,11 +35,16 @@ export async function create(html: string, options?: CreateOptions): Promise<Cre
...
-    chrome = await launchChrome(myOptions);
+
+  async function getChrome() {
+    if (!myOptions.host && !myOptions.port) {
+      await throwIfCanceled(myOptions);
+      return launchChrome(myOptions);
+    }
   }
 
+  const chrome: LaunchedChrome | undefined = await getChrome();
```
Comment (graingert) [inline src/index.ts]: this type should be inferred, tbh.

Comment (westy92) [reply src/index.ts]: Let's pop an explicit return type on `getChrome()` and infer here.

File: test/index.ts [hunk 2]
```diff
@@ -511,21 +511,12 @@ describe('HtmlPdf', () => {
         expect(stream).to.be.an.instanceOf(Readable);
       });
 
-      it('should output a valid Stream', (done) => {
+      it('should output a valid Stream', async () => {
```
Comment (westy92) [inline test/index.ts]: Nice!

File: test/index.ts [hunk 3]
```diff
@@ -53,10 +54,9 @@ describe('HtmlPdf', () => {
     });
 
     it('should handle a Chrome launch failure', async () => {
-      let launchStub: sinon.SinonStub;
+      const launchStub = sinon.stub(chromeLauncher, 'launch').callsFake(() => Promise.reject(error));
```
Comment (westy92) [inline test/index.ts]: Let's land #63 and then reevaluate what to do here.

**Pull request closed without merging**